addons/web/static/tests/core/notifications/notifications_tests.js

```diff
@@ -68,7 +68,7 @@ QUnit.test("notification with messageIsHtml option", async (assert) => {
     const notifService = env.services.notification;
     await mount(NotificationContainer, { env, target, props });
 
-    notifService.add("<i>Some message</i>", { messageIsHtml: true });
```

Comment:
nope, you're undoing what prettier enforces (also, I suggest you to enable the scripts if you have to modify non legacy files in web)

Reply:
Well spotted, that's really just a mistake in reversing the patch: I originally changed it thinking it was a test of the "legacy" notifications, but it turned out to be the opposite and I missed that it still diverged from the original after hand-reverting it.